### PR TITLE
v0.13.2 changes

### PR DESCRIPTION
**Pull request by foxish**
## Change description

New changes for v0.13.2

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: signadot/operator/templates/resources.signadot.com-customresourcedefinition.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ metadata:
     {{- range $key, $val := .Values.commonAnnotations }}
     {{ $key | quote }}: {{ $val | quote }}
     {{- end }}
+  creationTimestamp: null
```
Comment (foxish) [inline signadot/operator/templates/resources.signadot.com-customresourcedefinition.yaml:10]: Hmm, wondering why this change crept in - any dependency of the updater script?

Comment (scott-cotton) [reply signadot/operator/templates/resources.signadot.com-customresourcedefinition.yaml:10]: I don't think it is a function of the change to hack/templatize.go.  I would guess it's some toolchain version diff.

**Review by scott-cotton — APPROVED** [review]
LGTM
